lib/fizzy/execute.cpp

```diff
@@ -585,39 +584,20 @@ T fnearest(T value) noexcept
 {
     static_assert(std::is_floating_point_v<T>);
 
-    // This seems not to be needed for the tested implementations of std::nearbyint(),
-    // but C/C++ specification for this function does not guarantee returning arithmetic NaN.
     if (std::isnan(value))
         return std::numeric_limits<T>::quiet_NaN();  // Positive canonical NaN.
 
-    // This implementation uses std::nearbyint() which rounds to integer using current rounding
-    // direction. The rounding direction is temporarily set to "to nearest" for the case when
-    // the current rounding direction is different.
+    // Check if the input integer (as floating-point type) is even.
+    // There is a faster way of doing that by bit manipulations of mantissa and exponent,
+    // but this one is compact and this function is rarely called.
+    static constexpr auto is_even = [](auto i) noexcept { return std::fmod(i, 2.0f) == 0; };
```

Comment:
`std::fmod` here can be one of two overloaded functions depending on `T`, right?

Maybe for less confusion
```suggestion
    static constexpr auto is_even = [](auto i) noexcept { return std::fmod(i, T{2.0}) == 0; };
```
but not sure if it's better

Reply:
Changed to `T{2}` and `T{0}`. The usage of `{}` guarantees lossless conversion.